<launch>
  <node name="semantical_map_generator.py" pkg="semantical_slam"
	type="semantical_map_generator" args="">
  </node>

  <node name="bt_adapter.py" pkg="semantical_slam"
	type="bt_adapter" args="">
  </node>

  <node name="ir_adapter.py" pkg="semantical_slam"
	type="bt_adapter" args="">
  </node>
   
</launch>
